src/inject/inject.js

```diff
@@ -1,51 +1,89 @@
-chrome.extension.sendMessage({}, function(response) {
-  var readyStateCheckInterval = setInterval(function() {
-    if (document.readyState === "complete") {
-      clearInterval(readyStateCheckInterval);
+(function() {
+  let elementsWithZLCC = [];
 
-      // Check Page
-      checkPage();
+  const checkPage = function() {
+    const allElements = document.getElementsByTagName('*');
+    [...allElements].forEach( checkElement );
+  }
 
-      // Check page again when any input field is changed
-      const inputs = document.querySelectorAll('input');
-      [...inputs].forEach( function( input ) {
-        input.addEventListener( 'change', checkPage );
-      });
+  // From: https://jsfiddle.net/tim333/np874wae/13/
+  const checkElement = function( element ) {
+    const text = textWithoutChildren( element );
 
-    }
-  }, 10);
+    [...text].forEach( function( character ) {
+      unicodeCode = character.codePointAt(0);
 
-});
+      if (
+        zeroLengthCharacterCodes.includes( unicodeCode )
+        && !elementsWithZLCC.includes(element)
+      ) {
+        elementsWithZLCC.push(element)
+      }
+    });
 
-const checkPage = function() {
-  const allElements = document.getElementsByTagName('*');
-  [...allElements].forEach( checkElement );
-}
+    elementsWithZLCC.map(function( element ) {
```

Comment:
Should we use `forEach` rather than `map` as we aren't returning the values?

Reply:
You are right!